### DISCUSSION TITLE
TuYa SPM02 negative power

### DISCUSSION THREAD

**Discussion by bluettino** [Koenkk/zigbee2mqtt#19819] [Devices]
hello everyone, I have a spm02 meter, installed upstream of the main switchboard on a 3-phase system. I also have a photovoltaic system which is also 3-phase. The energy injected and withdrawn is correctly indicated by the 2 registers, but the power is always only indicated in absolute value and never as positive if absorbed and negative if fed into the grid. I use zigbee2mqtt and homeassistant..is it normal that the sign is not indicated ? 

thank you very much for any clarification

**Comment by defaktor** [comment]
Hi, do you have any news about your problem? I have the same situation with two devices.

**Comment by defaktor** [comment]
I think I'll find a solution. In my case, I connect the monitor before the RCD, and I connect the phase wires after the RCD; when they were reconnected in front of the RCD, everything returned to normal.
